docs/developer-docs/integrations/multi-chain/ckbtc-faq.md

```diff
@@ -0,0 +1,388 @@
+
+4. **Trustless Process**: The conversion of ckBTC back to BTC and vice versa is controlled at the Internet Computer protocol-level. It is accessible to all ICP decentralized applications (dapps), ensuring that no central authority has the power to restrict or censor these conversions.
+
+5. **Decentralized Process**: Every step of converting BTC to ckBTC and back is decentralized, meaning there are no centralized custodians, no bridges and no traditional cloud providers that could act as attack vectors.
+
+6. **1:1 Pegging**: Securing ckBTC 1:1 with BTC is crucial as it eliminates the risk of depegging.
+
+7. **Bitcoin Integration at Protocol Level**: ckBTC on the Internet Computer is part of a native integration with the Bitcoin network at the protocol level. This means that transactions occur directly between the two networks without any intermediary bridges.
+
+: [Taking security seriously: two top ICP features assessed by Trail of Bits](https://medium.com/dfinity/taking-security-seriously-two-top-icp-features-assessed-by-trail-of-bits-4a0023ab1e68#8e43)
+: [What you need to know about Bitcoin integration and ckBTC](https://internetcomputer.org/bitcoin-integration/faq#bitcoin-integration)
+: [Trustless](https://medium.com/dfinity/how-ckBTC-solves-the-dilemma-of-blockchain-bridges-ee8e0b72ee59#7ee8)
+: [Architecture Part 2](https://wiki.internetcomputer.org/wiki/Chain-key_Bitcoin#firstHeading)
+: [No Middle Person](https://medium.com/dfinity/chain-key-bitcoin-a-decentralized-bitcoin-twin-ceb8f4ddf95e#99f9)
+: [How ckBTC Solves the Dilemma of Blockchain Bridges Part 1](https://medium.com/dfinity/how-ckBTC-solves-the-dilemma-of-blockchain-bridges-ee8e0b72ee59#6164)
+
+## Can I trust that my funds are safe when using ckBTC?
+
+Yes, you can trust that your funds are safe when using ckBTC. Here's why:
+
+1. **Trustless Bitcoin Integration**: ckBTC uses a cryptographic technique that allows each smart contract canister to generate its own public key. These smart contracts can directly interact with the Bitcoin blockchain and send, receive, and hold Bitcoins. The conversion of ckBTC back to BTC and vice versa is controlled at the Internet Computer protocol-level, ensuring that no central authority has the power to restrict or censor these conversions. All transactions are transparent and can be verified through a real-time dashboard.
+
+2. **KYT Checks**: The ckBTC minter interacts with the KYT canister for "know-your-transaction" (KYT) checks. These checks are meant to ensure that the ckBTC minter only uses "clean" bitcoins to back the issued ckBTC tokens and to prevent transferring bitcoins to Bitcoin addresses that are considered to be associated with illicit activity. Cross-chain transactions go through KYT checks to protect honest users.
```

Comment:
```suggestion
The ckBTC minter interacts with the KYT canister for "know-your-transaction" (KYT) checks. These checks are meant to ensure that the ckBTC minter only uses "clean" bitcoins to back the issued ckBTC tokens and to prevent transferring bitcoins to Bitcoin addresses that are considered to be associated with illicit activity. Cross-chain transactions go through KYT checks to protect honest users.
```